scripts/push-docker.sh

```diff
@@ -7,12 +7,25 @@ set -euox
 
 echo $DOCKERHUB_TOKEN | docker login --username $DOCKERHUB_USERNAME --password-stdin
 
-if [ "$TAG" = "master" ] || [ "$TAG" = "main" ]; then
-    TAG=latest
+if [ "$COMMIT_SHA" = "" ]; then
+    echo "COMMIT_SHA was not provided"
+    exit 1
 fi
 
-docker tag mattermost/mattermost-cloud:test mattermost/mattermost-cloud:$TAG
-docker tag mattermost/mattermost-cloud-e2e:test mattermost/mattermost-cloud-e2e:$TAG
+echo "Tagging images with SHA $COMMIT_SHA"
 
-docker push mattermost/mattermost-cloud:$TAG
-docker push mattermost/mattermost-cloud-e2e:$TAG
+docker tag mattermost/mattermost-cloud:test mattermost/mattermost-cloud:$COMMIT_SHA
+docker tag mattermost/mattermost-cloud-e2e:test mattermost/mattermost-cloud-e2e:$COMMIT_SHA
+
+docker push mattermost/mattermost-cloud:$COMMIT_SHA
+docker push mattermost/mattermost-cloud-e2e:$COMMIT_SHA
+
+if [ "$REF_NAME" = "master" ] || [ "$REF_NAME" = "main" ]; then
```

Comment:
Shouldn't this be `==`?

```suggestion
if [ "$REF_NAME" == "master" ] || [ "$REF_NAME" == "main" ]; then
```

Reply:
For single [, we can use =. For [[, we can use ==

Reply:
Yeah, this is my understanding as well. For the record I hate shell scripting and could be wrong.

Reply:
https://linuxize.com/post/how-to-compare-strings-in-bash/#comparison-operators

Reply:
My bash is a bit rusted as well, that's why I wasn't 100% sure.

Reply:
Was chcking more information about it: https://unix.stackexchange.com/a/306115